google/cloud/aiplatform/pipeline_jobs.py

```diff
@@ -67,13 +69,26 @@ def _set_enable_caching_value(
           Required. The dictionary of pipeline spec.
      enable_caching (bool):
           Required. Whether to enable caching.
+     cache_components (List[str]):
```

Comment:
`cache_components` was optional in the PipelineJob instantiation phase, but required here. Could you update this to be optional? Otherwise it would break the existing users.